tests/components/renault/snapshots/test_sensor.ambr

```diff
@@ -1577,70 +1577,6 @@
       'unique_id': 'vf1aaaaa555777999_hvac_last_activity',
       'unit_of_measurement': None,
     }),
-    EntityRegistryEntrySnapshot({
-      'aliases': set({
-      }),
-      'area_id': None,
-      'capabilities': None,
-      'config_entry_id': <ANY>,
```

Comment:
why are these snapshots deleted?

Reply:
"res" endpoint is now correctly marked as not-supported for zoe40 ... so it no longer creates the unavailable entities